docs/index.md

```diff
@@ -32,99 +24,44 @@ The workflow of using ABR is held as simple as possible.
 
 As outlined above, each scene requires what we call a *backend implementation*. 
 This implementation takes care of loading a blender file, setting up everything that 
 is required such as camera information, objects, randomization, etc.
 It also contains a `main loop` which does the rendering for the number of desired images. 
 An exemplary backend implementation can be found in ABR source tree at
-``src/amira_blender_rendering/scenes/workstationscenarios.py``.
+`src/amira_blender_rendering/scenes/workstationscenarios.py`.
 This backend implementation reads all optional configuration parameters either
 from a configuration file that is passed along to the rendering script, or from
 the additional parameters passed during execution.
 
 An example for a configuration that contains documentation for all options can
-be found in ABR source tree at ``config/examples/workstation_scenario01_test.cfg`` .
+be found in ABR source tree at `config/examples/workstation_scenario01_test.cfg` .
 Note that configuration options depend on the specified blender scene and backend
 implementation.
 
-Find out more about how to use ABR in :doc:`using` and Configuration :doc:configs/overview`.
+Find out more about [how to use ABR](./using.md) and [Configuration](./configs/overview.md).
 
+## Citing ABR
 
-Citing ABR
-----------
 If you use ABR please make sure to cite our work.
 
-.. code-block:: latex
-
-   @misc{amira_blender_rendering_2020,
-       author={N.Waniek, M.Todescato, M.Spies, M.Buerger},
-       title={AMIRA Blender Rendering},
-       year={2020},
-       url={https://github.com/boschresearch/amira-blender-rendering},
-   }
-
-
-
-.. toctree::
-   :maxdepth: 3
-   :caption: ABR User's Guide
-
-   installation.rst
-   using.rst
-   formats.rst
-   datasets.rst
-   fqa.rst
-   troubleshooting.rst
-   tests.rst
-   license.rst
-
-.. toctree::
-   :maxdepth: 3
-   :caption: ABR Developer's Guide
-
-   contributing.rst
-
-.. toctree::
-   :maxdepth: 3
-   :caption: Configurations
-   :name: configurations
-
-   configs/overview.rst
-   configs/baseconfiguration.rst
-   configs/workstation_scenarios.rst
-   configs/pandatable.rst
-   configs/simpleobject.rst
-   configs/multiview.rst
-
-.. toctree::
-  :maxdepth: 3
-  :caption: Datasets
-
-
-.. toctree::
-  :maxdepth: 3
-  :caption: Tutorials
-
-  tutorials/simplecustomscenario.rst
-
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
-
+```latex
+@misc{amira_blender_rendering_2020,
+    author={N.Waniek, M.Todescato, M.Spies, M.Buerger},
+    title={AMIRA Blender Rendering},
+    year={2020},
+    url={https://github.com/boschresearch/amira-blender-rendering},
+}
+```
 
-Contacts
-========
+## Contacts<a name="contacts"></a>
 
-| `Nicolai Waniek <mailto:Nicolai.Waniek@de.bosch.com>`_
-| `Marco Todescato <mailto:Marco.Todescato@de.bosch.com>`_
-| `Markus Spies <mailto:Markus.Spies2@de.bosch.com>`_
+- [Nicolai Waniek](mailto:Nicolai.Waniek@de.bosch.com)
```

Comment:
@nwaniek what do you want us to do with you email address?

Reply:
You could change it to n@rochus.net, thanks!

Reply:
great... I think we also need to update the NOTICE file stating your new affiliation if you are going to contribute in the future (which I hope :-D)

Reply:
Yes, I'll contribute soon-ish! The PR is almost ready, but I couldn't work and won't be able to work on it for some time: desktop PC is in a box waiting to be transported, and my notebook is unsuitable for developing and debugging blender extensions ;)

About my affiliation: You can either leave it off entirely, or simply add "Kavli Institute for Systems Neuroscience"

Reply:
perfect! will do!